app/src/main/java/com/dgaf/happyhour/Model/Adapter/DealListAdapter.java

```diff
@@ -213,6 +224,9 @@ public void applyDayOfWeekForQuery(ParseQuery<DealModel> query) {
                 query.whereGreaterThanOrEqualTo("sundaySt",0);
                 query.whereLessThanOrEqualTo("sundayEn",2400);
                 break;
+            case TODAY:
+                //TODO filter todays deals after current time
+                break;
```

Comment:
I think this is important functionality, but not sure what it would look like.  We should discuss this more at a team meeting.